opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -501,6 +530,50 @@ def use_span(
             end_on_exit: Whether to end the span automatically when leaving the
                 context manager.
         """
+
+
+class DefaultTracer(Tracer):
+    """The default Tracer that is used when no Tracer implementation is available.
+
+    All operations are no-op.
+    """
+
+    # Constant used to represent the current span being used as a parent.
+    # This is the default behavior when creating spans.
+    CURRENT_SPAN = DefaultSpan(INVALID_SPAN_CONTEXT)
```

Comment:
Why not use `Tracer.CURRENT_SPAN`?

Reply:
Just realized this is actually not needed here. Removing